README.md

```diff
@@ -87,7 +103,13 @@ Example of using the [Referrers API](https://github.com/opencontainers/distribut
     trust_store: <dir_to_user_trust_store>
     allow_referrers_api: <boolean_flag_for_referrers_api>
 ```
-For example,
+
+<details>
+
+<summary>See an example (Click here).</summary>
+
+`.github/trustpolicy/trustpolicy.json` MUST follow the Notation [trust policy specs](https://github.com/notaryproject/specifications/blob/v1.0.0-rc.2/specs/trust-store-trust-policy.md#trust-policy).
```

Comment:
We should update the link to point to `v1.0.0` instead of a rc version.
```suggestion
`.github/trustpolicy/trustpolicy.json` MUST follow the Notation [trust policy specs](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/trust-store-trust-policy.md#trust-policy).
```